integration/setup/client.go

```diff
@@ -106,12 +106,14 @@ func setupClient(tb testing.TB, ctx context.Context, uri string) *mongo.Client {
 	defer trace.StartRegion(ctx, "setupClient").End()
 
 	client, err := makeClient(ctx, uri)
-	require.NoError(tb, err, "URI: %s", uri)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I don't think we should assert on some errors and return others. The simplest solution might be panic

Reply:
Ok, if panic is enough, let's panic.

Reply:
Actually, there is a simpler fix